backend/experiment/rules/toontjehoger_4_absolute.py

```diff
@@ -199,15 +199,16 @@ def validate_playlist_groups(self, groups):
         # Check if the groups are sequential and unique
         integer_groups.sort()
         if integer_groups != list(range(1, len(groups) + 1)):
-            return ['Groups in playlist sections should be sequential numbers starting from 1 to the number of sections in the playlist ({}). E.g. "1, 2, 3, ... {}"'.format(len(groups), len(groups))]
+            return ['Groups in playlist sections should be sequential numbers starting from 1 to the number of items in the playlist ({}). E.g. "1, 2, 3, ... {}"'.format(self.PLAYLIST_ITEMS, self.PLAYLIST_ITEMS)]
 
         return []
 
     def validate_playlist(self, playlist: Playlist):
         errors = super().validate_playlist(playlist)
 
         # Get group values from sections, ordered by group
-        groups = list(playlist.section_set.values_list('group', flat=True))
+        groups = list(playlist.section_set.values_list(
+            'group', flat=True).distinct())
```

Comment:
This is the actual needed change though - before, there were as many groups as there were sections. I still think the reference to `PLAYLIST_ITEMS` n the `validate_playlist_groups` function is more readable.